### DISCUSSION TITLE
MWL in dcm4che toolkit

### DISCUSSION THREAD

**Discussion by amaljesudas** [dcm4che/dcm4che#1176] [Q&A] [answered]
I need to execute a server from where I can download worklists.
Am using dcm4che-5.25.1.
Just roughly went through some of the tools, but could not identify any that seemed to offer this.
Does dcm4che tool kit have such a support?

Thanks in advance.

**Comment by SamuelImriska** [comment, marked as answer]
Sorry but there is currently no such tool in dcm4che toolkit.

**Reply by amaljesudas** [reply]
Thanks for the confirmation @SamuelImriska
